src/mlpack/tests/split_data_test.cpp

```diff
@@ -291,34 +334,45 @@ TEST_CASE("StratifiedSplitLargerDataResultTest", "[SplitDataTest]")
 {
   mat input(3, 480);
   input.randu();
-
```

Comment:
Any reason to remove these blank lines?  I think it was a bit easier to read before.

Reply:
It might had happened due to the code editor. Fixed now.